rconweb/api/views.py

```diff
@@ -71,27 +72,31 @@ def set_name(request):
     )
 
 
+# TODO: this is not an exposed endpoint
 @csrf_exempt
-@login_required(True)
+@login_required()
 @record_audit
 def set_temp_broadcast(request):
     return set_temp_msg(request, temporary_broadcast, "set_temp_broadcast")
 
 
+# TODO: this is not an exposed endpoint
 @csrf_exempt
-@login_required(True)
+@login_required()
 @record_audit
 def set_temp_welcome(request):
     return set_temp_msg(request, temporary_welcome, "set_temp_welcome")
 
 
 @csrf_exempt
+@permission_required("api.can_view_crcon_version", raise_exception=True)
 def get_version(request):
     res = run(["git", "describe", "--tags"], stdout=PIPE, stderr=PIPE)
     return api_response(res.stdout.decode(), failed=False, command="get_version")
 
 
 @csrf_exempt
+@permission_required("api.can_view_public_info", raise_exception=True)
 def public_info(request):
```

Comment:
This is public info, you don't need a login. The stats page and the score bot use that. You'll break stuff I think

Reply:
Whoops, fixed.